src/components/SideMenu/SideMenu.js

```diff
@@ -47,33 +37,33 @@ class SideMenu extends Component {
           </div>
         </span>
 
-        <div className="sidemenu__nav" style={{ width: sidemenu }}>
+        <div className="sidemenu__background" onClick={() => this.closeSideMenu()} style={{ display: background }}></div>
+        <div className="sidemenu__nav" style={{ transform: sidemenu }}>
```

Comment:
@nicholasmiller1 Here's a picky note for next time: you should be more descriptive with this variable. ie, `sidemenuTranslate` instead of `sidemenu`